CONTRIBUTING.md

```diff
@@ -64,6 +64,12 @@ In order to run terraform or aws cli etc., just run the below and enter the MFA
 eval $(assume-role gdx)
 ```
 
+#### Grafana access
+
```

Comment:
Can we link to this from the monitoring doc?

Reply:
Done